scene/resources/animation.cpp

```diff
@@ -3849,8 +3951,16 @@ void Animation::_bind_methods() {
 	BIND_ENUM_CONSTANT(LOOP_LINEAR);
 	BIND_ENUM_CONSTANT(LOOP_PINGPONG);
 
+#ifdef TOOLS_ENABLED
 	BIND_ENUM_CONSTANT(HANDLE_MODE_FREE);
+	BIND_ENUM_CONSTANT(HANDLE_MODE_LINEAR);
 	BIND_ENUM_CONSTANT(HANDLE_MODE_BALANCED);
+	BIND_ENUM_CONSTANT(HANDLE_MODE_MIRRORED);
+
+	BIND_ENUM_CONSTANT(HANDLE_SET_MODE_NONE);
+	BIND_ENUM_CONSTANT(HANDLE_SET_MODE_RESET);
+	BIND_ENUM_CONSTANT(HANDLE_SET_MODE_AUTO);
+#endif // TOOLS_ENABLED
```

Comment:
This is the same problem as for the methods, there shouldn't be `TOOLS_ENABLED` only API exposed to bindings. These enums values are not useful to users so they shouldn't be part of the public API.

Reply:
Removed these.